assets/dev/js/frontend/handlers/container/grid-outline.js

```diff
@@ -0,0 +1,229 @@
+export default class GridOutline extends elementorModules.frontend.handlers.Base {
+	isActive() {
+		return elementorFrontend.isEditMode();
+	}
+
+	getDefaultSettings() {
+		return {
+			selectors: {
+				gridOutline: '.e-grid-outline',
+				directGridOverlay: ':scope > .e-grid-outline',
+				boxedContainer: ':scope > .e-con-inner',
+			},
+			classes: {
+				outline: 'e-grid-outline',
+				outlineItem: 'e-grid-outline-item',
+			},
+		};
+	}
+
+	getDefaultElements() {
+		const selectors = this.getSettings( 'selectors' );
+
+		return {
+			outlineParentContainer: null,
+			gridOutline: this.findElement( selectors.gridOutline ),
+			directChildGridOverlay: this.findElement( selectors.directGridOverlay ),
+		};
+	}
+
+	onInit() {
+		super.onInit();
+
+		this.initLayoutOverlay();
+	}
+
+	bindEvents() {
+		elementorFrontend.elements.$window.on( 'resize', this.onDeviceModeChange.bind( this ) );
+		this.addChildLifeCycleEventListeners();
+	}
+
+	unbindEvents() {
+		this.removeChildLifeCycleEventListeners();
+		elementorFrontend.elements.$window.off( 'resize', this.onDeviceModeChange.bind( this ) );
+	}
+
+	initLayoutOverlay() {
+		this.getContainer();
```

Comment:
@davseve This function name is a bit confusing? It sounds like it needs to `return` a container element. 
It is more like this is a `setter` and not a `getter`.
Can you improve the readability in this way perhaps:
```
initLayoutOverlay() {
    if ( ! this.shouldDrawOutlines() ) {
        return;
    }
    ...
}
```

Reply:
`getContainer`  is a void function, it doesn't return anything
I have replaced the function name from `getContainer` to `shouldDrawOutlines`.

Reply:
Fixed, something happened in the push process.
Now it looks up to date.
Sorry for the inconvenience.